src/home/Home.tsx

```diff
@@ -1,36 +1,30 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
+import { useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
-import useSticky from '../common/hooks/useSticky';
+import { useSticky } from '../common/hooks/useSticky';
 import { SocialMediaBar } from '../components/SocialMediaBar/SocialMediaBar';
 import { SubscriptionBox } from '../components/SubscriptionBox/SubscriptionBox';
 import { authRoutes } from '../routes';
+import { AppState } from '../store/app/types';
+import { State } from '../store/types';
 import { useHomeStyles } from './Home.styles';
 
 const ParalaxHeader: React.FC = () => {
   const classes = useHomeStyles();
-  const [offset, setOffset] = useState(0);
-  const parallaxShift = () => {
-    setOffset(window.pageYOffset);
-  };
-
-  const { sticky } = useSticky();
-
-  useEffect(() => {
```

Comment:
I combined this useEffect with the other since they are doing pretty much the same thing